sherpa/astro/ui/utils.py

```diff
@@ -8635,18 +8637,25 @@ def fake_pha(self, id, arf, rmf, exposure, backscal=None, areascal=None,
            dataset identifier is used, rather than creating a
            completely new dataset.
 
+        .. versionchanged:: 4.13.2
+           Multiple ARFs and RMFs can now be used for the simulation.
+           Furthermore, background models can be specified instead of
+           background `~sherpa.stro.data.DataPHA` datasets.
+
         Parameters
         ----------
         id : int or str
            The identifier for the data set to create. If it already
            exists then it is assumed to contain a PHA data set and the
            counts will be over-written.
-        arf : filename or ARF object
+        arf : filename or ARF object or list of filenames
```

Comment:
I have two related questions here as we look at the `fake_pha` command:

1. I dislike the way it over-writes my existing dataset and would prefer it if it creates a new one - I think the ship has long sailed on that and we should just make sure we can easily "copy" a dataset (there is `copy_data` which does this, I just don't know if it also copies responses or just points to the same response).
2. if I have a PHA file loaded with arf and rmf why should I give the arf and rmf, why not just use them

I think of fake_pha as either being used either for a proposal - in which case you are not necessarily going to have a loaded-PHA file to use - or to see how significant a feature or model is for an observation (when you probably have everything set up in the PHA file when it's loaded). Do these two use cases have "different" interfaces? Probably not.

Reply:
Well, "the ship has sailed". My change here is actually incremental. You could always run a `fake_pha` with either a filename or an arf object. I did not change that. What I did change, is the ability to pass in a list of filenames (and a list of ARF objects should also be possible. Need to check if I put that in the code and just forgot it for the docs or not. In the same vain, point (2) is a change to the existing UI that goes beyond just adding the ability to run multiple ARF/RMF. In the past, you could use an id that was taken and it would just be overwritten with the new ARF and RMF. The same happens here.

I like the suggestion to reuse ARFs and RMFs is present (say with "arf=None", which could be the default), but I'm not sure we want roll this into this PR or leave it for a follow-up since it's a separate issue really.

Reply:
done.